ddp-workspace/projects/ddp-sdk/src/lib/components/languageSelector.component.ts

```diff
@@ -0,0 +1,111 @@
+import { Component, Inject, Input, OnDestroy, OnInit } from "@angular/core";
+import { StudyLanguage } from "../models/studyLanguage";
+import { ConfigurationService } from "../services/configuration.service";
+import { LanguageService} from "../services/languageService.service";
+import { LanguageServiceAgent } from "../services/serviceAgents/languageServiceAgent.service";
+import { isNullOrUndefined } from "util";
+import { Subscription } from "rxjs";
+
+@Component({
+  selector: 'ddp-language-selector',
+  template: `
+    <div [hidden]="!loaded">
+      <button class="SimpleButton" [ngClass]="{'SimpleButton--Scrolled': isScrolled}" *ngIf="currentLanguage !== null && currentLanguage !== undefined" [matMenuTriggerFor]="menu">
+        <svg class="ddp-globe" height="24px" width="24px">
+          <title id="title" [lang]="currentLanguage.languageCode" translate>Toolkit.Header.LanguageSelection</title>
```

Comment:
One small issue: translate for SDK components we should store in SDK section in the en.json

Reply:
Sorry, I've noticed it too late.

Reply:
Fixed it!